src/ai/backend/storage/context.py

```diff
@@ -39,6 +39,7 @@
 )
 from .purestorage import FlashBladeVolume
 from .types import VolumeInfo
+from .vast import VastVolume
```

Comment:
We need to call them `VAST`, not `Vast` ([link to their official webpage](https://vastdata.com/)). Capital letters do matter!